components/script/dom/htmlmediaelement.rs

```diff
@@ -1279,12 +1282,32 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
     // https://html.spec.whatwg.org/multipage/#dom-media-autoplay
     make_bool_setter!(SetAutoplay, "autoplay");
 
+    // https://html.spec.whatwg.org/multipage/#dom-media-defaultmuted
+    make_bool_getter!(DefaultMuted, "muted");
+    // https://html.spec.whatwg.org/multipage/#dom-media-defaultmuted
+    make_bool_setter!(SetDefaultMuted, "muted");
+
     // https://html.spec.whatwg.org/multipage/#dom-media-src
     make_url_getter!(Src, "src");
 
     // https://html.spec.whatwg.org/multipage/#dom-media-src
     make_url_setter!(SetSrc, "src");
 
+    // https://html.spec.whatwg.org/multipage/#dom-media-muted
+    fn Muted(&self) -> bool {
+        self.muted.get() as bool
+    }
+
+    // https://html.spec.whatwg.org/multipage/#dom-media-muted
+    fn SetMuted(&self, value: bool) {
+        self.muted.set(value);
```

Comment:
Apart from setting the value of the `mute` property, we need to set the same value in the media backend. To do that, we need to expose a new method in the servo-media [Player](https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gst-plugins-bad-libs/html/GstPlayer.html#gst-player-set-mute) trait (i.e. `set_mute`) which [implementation](https://github.com/servo/media/blob/master/backends/gstreamer/src/player.rs#L616) should use the [GStreamer `set_mute`](https://slomo.pages.freedesktop.org/rustdocs/gstreamer/gstreamer_player/struct.Player.html#method.set_mute) method.